cpp/test/DataStorm/partial/Test.ice

```diff
@@ -4,7 +4,7 @@
 module Test
 {
 
-class Stock
+struct Stock
```

Comment:
replaced with a struct, because class by value doesn't work. I think the replacement is fine here as the test is about partial updates not about using Ice classes.